pkg/status/status.go

```diff
@@ -55,7 +55,7 @@ type status struct {
 	coAPINotPresent bool
 	namespace       string
 	clusterOperator *configv1.ClusterOperator
-	version         string
+	releaseVersion  string
```

Comment:
Isn't version here being the release version obvious? Why do you need to call it out and make the code verbose?

Reply:
I had the same thought at first, but then realized 'releaseVersion` makes the purpose of the variable more clear rather than just `version`. `version` isn't exactly obvious for someone reading the code for the first time and has to dig around a little extra to understand the meaning of the variable. 
Besides, if we have variables names `coAPINotPresent` or `clusterOperator`, `releaseVersion` doesn't seem that verbose in comparison.

Reply:
Having other verbose variables doesn't mean we can keep adding more. 

What other purpose does version have? If you want to be technically correct then it should be `currentVersion`. I would rather leave it as `version` because it obviously means the version of the operator that you are using.

Reply:
> Having other verbose variables doesn't mean we can keep adding more.

+1,000,000

Reply:
I believe that the release version of CVO and the version of marketplace are conceptually different things. Example: CVO could have multiple release version that reference the same version of marketplace.

Reply:
In this context it is the version of marketplace. It just so happens we have decided to use the release version specified by CVO. PS: this is not the release version of CVO but the version specified by it.

Reply:
Understood - I will undo the `releaseVersion` changes.